Packs/AggregatedScripts/Scripts/FileEnrichmentV2/FileEnrichmentV2.py

```diff
@@ -0,0 +1,142 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+from AggregatedCommandApiModule import *
+
+INTERNAL_ENRICHMENT_BRANDS = ["WildFire-v2"]
+FILE_CONTEXT_PATH = "FileEnrichmentV2(val.MD5 && val.MD5 == obj.MD5 || val.SHA1 && val.SHA1 == obj.SHA1 || ' \
```

Comment:
Didn't we change this one? it will not give you to have the same hashes in different forms.